lib/matplotlib/tests/__init__.py

```diff
@@ -1,10 +1,13 @@
+# Check for the matplotlib_baseline_images.
 from pathlib import Path
 
 
-# Check that the test directories exist.
-if not (Path(__file__).parent / 'baseline_images').exists():
-    raise IOError(
-        'The baseline image directory does not exist. '
-        'This is most likely because the test data is not installed. '
-        'You may need to install matplotlib from source to get the '
-        'test data.')
+try:
+    import matplotlib_baseline_images
+except:
+    if not (Path(__file__).parent / 'baseline_images').exists():
+        raise ImportError(
+            'The baseline image directory does not exist. '
+            'This is most likely because the test data is not installed and baseline image directory is absent. '
+            'You may need to install matplotlib_baseline_images to get the '
+            'test data.')
```

Comment:
This error message should detailed instructions of how to fix the problem (either by installing or generating).